_integration-tests/tests/go-redis.v8/main.go

```diff
@@ -10,33 +10,68 @@ import (
 	"fmt"
 	"log"
 	"net/http"
+	"net/url"
 	"orchestrion/integration"
+	"os"
 	"time"
 
+	"github.com/testcontainers/testcontainers-go"
+	testredis "github.com/testcontainers/testcontainers-go/modules/redis"
+	"github.com/testcontainers/testcontainers-go/wait"
+
 	"github.com/go-redis/redis/v8"
 )
 
 func main() {
+	if os.Getenv("DOCKER_NOT_AVAILABLE") != "" {
+		log.Println("Docker is required to run this test. Exiting with status code 42!")
+		os.Exit(42)
+	}
+
+	ctx := context.Background()
+	server, err := testredis.RunContainer(ctx,
+		testcontainers.WithImage("redis:7"),
```

Comment:
It is not the redis 8 test?

Reply:
The _driver_ is v8, but Redis v7 is the latest ^^